src/makeRobotAccountant.js

```diff
@@ -7,6 +7,27 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 1;
+
+  return function(x) {
+    const copyX = x;
+
+    return function(y) {
+      if (count <= 3) {
+        count++;
+
+        return copyX + y;
+      } else if (count > 3 && count % 2 === 1) {
+        count++;
+
+        return copyX + y;
+      } else {
+        count++;
```

Comment:
better use such pattern

```
if (...) { // if something goes wrong
  // we stop doing what we're doing
}

// otherwise – we do what we need

```

Reply:
ok

Reply:
а таку логіку ми маємо застосовувати конкретно для цієї задачі , чи для всіх ?